docs/platforms/flutter/integrations/routing-instrumentation.mdx

```diff
@@ -107,6 +100,64 @@ final GoRouter _router = GoRouter(
 );
 ```
 
+## Time to Initial Display
+
+Time to initial display (TTID) provides insight into how long it takes your Widget to launch and draw their first frame. This is measured by adding a span for navigation to a Widget. The SDK sets the span operation to `ui.load.initial-display` and the span description to the Widget's route name followed by initial display (for example, `MyWidget initial display`).
+There are two ways to set up TTID:
+
+### Approximation
+
+The time to initial display is enabled by default and provides an approximation of TTID.
```

Comment:
```suggestion
The default setup is enabled automatically, but only provides an approximation of TTID. To set a more accurate TTID, manually wrap the desired widget with `SentryDisplayWidget`, as shown below:
```

I think it makes sense to put these two sections together since they're both really short and the user has to choose between the two.